src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,806 @@
+     *              Split for each individual table: Switches, Links, and Interfaces.
+     */
+    changeSwitchData(data) {
+      this.currentSwitchData = data
+    },
+    changeLinkData(data) {
+      this.currentLinkData = data
+    },
+    changeInterfaceData(data) {
+      this.currentInterfaceData = data
+    },
+    /**
+     * @param {Object} item - The normal table row object containing:
+     * @param {string} item.dpid - The data path id of the device the Object references.
+     * @param {string} item.id - The id of the device the Object references.
+     * @param {string} index - The index of the current table row Object. Usually id or name.
+     * @description Adds and Removes elements from the respective "selected" list based on id.
+     *              Split for each individual table: Switches, Links, and Interfaces.
+     */
+    add_remove_Switches(item, index) {
+      if (this.selectedSwitches[index]?.dpid == item.dpid) {
+        delete this.selectedSwitches[index]
+      } else {
+        this.$set(this.selectedSwitches, index, item);
+      }
+      this.change = this.change * -1
+      this.$forceUpdate();
+    },
+    add_remove_Links(item, index) {
+      if (this.selectedLinks[index]?.id == item.id) {
+        delete this.selectedLinks[index]
+      } else {
+        this.$set(this.selectedLinks, index, item);
+      }
+      this.change = this.change * -1
+      this.$forceUpdate();
+    },
+    add_remove_Interfaces(item, index) {
+      if (this.selectedInterfaces[index]?.id == item.id) {
+        delete this.selectedInterfaces[index]
+      } else {
+        this.$set(this.selectedInterfaces, index, item);
+      }
+      this.change = this.change * -1
+      this.$forceUpdate();
+    },
+    //API Requests for Switch, Link, and Interface data.
+    get_topology() {
+      var self = this
+      let request = $.ajax({
+                       type:"GET",
+                       dataType: "json",
+                       url: this.$kytos_server_api + "kytos/topology/v3",
+                       async: true});
+      request.done(function(data) {
+        self.switches = data['topology']['switches']
+        self.links = data['topology']['links'];
+        self.interfaces = {}
+        for (var swid in self.switches) {
+          for (var intfid in self.switches[swid]["interfaces"]) {
+            self.interfaces[intfid] = self.switches[swid]["interfaces"][intfid];
+            self.interfaces[intfid]["interface_id"] = intfid;
+            self.interfaces[intfid]["dpid"] = swid;
+            self.interfaces[intfid]["content_switch"] = self.switches[swid];
+          }
+        }
+      });
+      request.fail(function(data) {
+        let notification = {
+             icon: 'cog',
+             title: 'Could not retrieve topology',
+             description: data.status.toString()
+        }
+        self.$kytos.$emit("setNotification", notification)
+      });
+    },
+  //   get_interfaces() {
```

Comment:
Can you remove this commented method? 

Looks like you meant to remove it, since you're fetching the entire topology.